Xamarin.Essentials/DeviceDisplay/DeviceDisplay.android.cs

```diff
@@ -34,15 +34,16 @@ static bool PlatformKeepScreenOn
         static DisplayInfo GetMainDisplayInfo()
         {
             using var displayMetrics = new DisplayMetrics();
-            using var display = GetDefaultDisplay();
+            var display = GetDefaultDisplay();
```

Comment:
Without this, it appears that Android 21-24 throws a ObjectDisposedException for the Display object